tests/templates/test_embeddings/test_amplitude.py

```diff
@@ -359,3 +359,49 @@ def test_torch(self, tol):
         res2 = circuit2(features)
 
         assert qml.math.allclose(res, res2, atol=tol, rtol=0)
+
+
+def test_multiple_amplitudes():
+    """Tests the use of many AmplitudeEmbedding."""
+
+    tol = 10e-10
+    dev = qml.device("default.qubit", wires=7)
+
+    @qml.qnode(dev)
+    def circuit():
+        qml.Hadamard(wires=0)
+        qml.templates.AmplitudeEmbedding([1, 1, 1, 1, 0, 0, 0, 0], wires=[1, 2, 3], normalize=True)
```

Comment:
Maybe it's worth checking if we have an operation in between two AmplitudeEmbedding operations.